### PR TITLE
DIP-159 Added readme description

### PR DESCRIPTION
**Pull request by cameron-white-janrain**

### CODE REVIEW

**Comment by RussellSmith-Janrain** [comment]
Looks good - thanks for adding this

File: README.md [hunk 1]
```diff
@@ -3,6 +3,66 @@
...
+- `REMOTE_PORT`: Port of the BlueKai SFTP Server.
+(default: `22`)
+
+- `REMOTE_USERNAME`: Username of the BlueKai SFTP Server.
+
+- `REMOTE_PASSWORD`: Password of the BlueKai SFTP Server.
+
+- `REMOTE_RSA_KEY`: RSA Private Key file of the BlueKai SFTP Server (Optional is using username and password)
+
+- `REMOTE_FILE`: The name to file to save on the BlueKai SFTP Server.
```
Comment (BCox-Janrain) [inline README.md]: `REMOTE_FILE` description is a little confusing. I'm assuming you meant "The name of the file to ..."

File: README.md [hunk 2]
```diff
@@ -3,6 +3,66 @@
...
+be kept. (default: `20`)
+
+- `AWS_DEFAULT_REGION`: AWS region the app runs in.
+
+- `AWS_DYNAMODB_URL`: Url of the DynamoDB service to use.
+(should only be used during local development, leave blank when deployed
+to elastic beanstalk)
+
+- `AWS_DYNAMODB_TABLE`: Name of the table in DynamoDB to use.
+(default: `import_service`)
```
Comment (BCox-Janrain) [inline README.md]: Does Bluekai use the DynamoDB? If it does, the default shouldn't be `import_service`

Comment (RussellSmith-Janrain) [inline README.md]: Isn't that just used for local?

File: README.md [hunk 3]
```diff
@@ -3,6 +3,66 @@
...
+- `JANRAIN_CLIENT_SECRET`: Secret for the client.
+
+- `JANRAIN_SCHEMA_NAME`: Name of the Capture schema containing the user records.
+(default: `user`)
+
+- `JANRAIN_BATCH_SIZE_LIMIT`: Max number of records to retrieve from Janrain
+at a time.
+(default: `1000`)
+
+- `JANRAIN_BATCH_SIZE_LIM`: Comma seperated list of schema attributes to export.
```
Comment (BCox-Janrain) [inline README.md]: `JANRAIN_BATCH_SIZE_LIM` should be `JANRAIN_ATTRIBUTE_KEYS`

File: README.md [hunk 4]
```diff
@@ -3,6 +3,66 @@
 [![Build Status](https://travis-ci.org/janrain/janrain-bluekai.svg?branch=master)](https://travis-ci.org/janrain/janrain-bluekai)
 [![Coverage Status](https://coveralls.io/repos/github/janrain/janrain-bluekai/badge.svg?branch=master)](https://coveralls.io/github/janrain/janrain-bluekai?branch=master)
 
+BlueKai is a cloud-based big data platform that enables companies to personalize online, offline, and mobile marketing campaigns. The service provides an `/export` endpoint which when POSTed to instructs the service to transfer new records from Janrain to a BlueKai SFTP server in a TSV file format. An external service is then reposible for importing the TSV file into BlueKai. The intent is to have a time based service, such as a CRON job, trigger the export by POSTing to the `/export` endpoint. The service is designed to be deployed in an AWS Elastic Beanstalk Application and is configured via environmental variables.
+
```
Comment (BCox-Janrain) [inline README.md]: Just some quick proofreading. last two words should be "environment variables."

File: README.md (hunk 2 above)
Comment (BCox-Janrain) [inline README.md]: Oh, missed this comment. I don't know if it's only used during local development, but I assume it uses DynamoDB on AWS. I need to read the DynamoDB python library docs to find out.

**Pull request merged**